packages/kit/tsconfig.enforced.json

```diff
@@ -8,6 +8,7 @@
 		"module": "es2020",
 		"moduleResolution": "node",
 		"allowSyntheticDefaultImports": true,
+		"types": ["./types"],
```

Comment:
This closes off all other ambient types from appearing, which might fail in the future. I'm also not sure why this suddenly is a problem.

Reply:
That might be a problem. Though, I really don't know why it suddenly decided to not detect the ambient types. The first 2 push seems to works just fine.